inference-engine/samples/speech_sample/main.cpp

```diff
@@ -708,10 +737,29 @@ int main(int argc, char *argv[]) {
 
         /** Stores all input blobs data **/
         std::vector<Blob::Ptr> ptrInputBlobs;
-        for (auto& input : cInputInfo) {
-            ptrInputBlobs.push_back(inferRequests.begin()->inferRequest.GetBlob(input.first));
+        if (!FLAGS_iname.empty()) {
+            std::string string_iname = FLAGS_iname;
+            std::vector<std::string> string_input;
+            size_t pos = 0;
+            while ((pos = string_iname.find(",")) != std::string::npos) {
+                string_input.push_back(string_iname.substr(0, pos));
+                string_iname.erase(0, pos + 1);
+            }
+            string_input.push_back(string_iname.substr(0, string_iname.length()));
+            for (std::string input : string_input) {
+                Blob::Ptr blob = inferRequests.begin()->inferRequest.GetBlob(input);
+                if (!blob) {
+                    std::string errMessage("The blob with name : "
+                                           + input + " was not recived ");
```

Comment:
"Recieved", not "recived". But "received" does not make sense here.
I would better formulate: "No blob with name: "